client/src/components/shared/Layout/LayoutTopBar/LayoutTopBar.tsx

```diff
@@ -76,16 +95,46 @@ const LayoutTopBar: FC<LayoutTopBarProps> = ({ className }) => {
       return;
     }
 
-    setIsSettingsDrawerOpen(true);
+    setShowSettingsDrawer(true);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isDesktop, pathname]);
 
   useEffect(() => {
-    if (isSettingsDrawerOpen && pathname !== ROOT_ROUTES.settings.absolute && !isDesktop) {
+    if (showSettingsDrawer && pathname !== ROOT_ROUTES.settings.absolute && !isDesktop) {
       navigate(ROOT_ROUTES.settings.absolute);
-      setIsSettingsDrawerOpen(false);
+      setShowSettingsDrawer(false);
```

Comment:
IMHO this change is for the worse and I would stick with previous naming.

`setShowSettingsDrawer` indicated to me this function sets `showSettingsDrawer`, which itself is an action.
`setIsSettingsDrawerVisible` indicates to me it sets `isSettingsDrawerVisible`, which is a state.

What is your take on that? Maybe there is some line I don't see here.

Reply:
I wanted to unify naming with other methods:
```
  setShowWalletModal,
  setShowConnectWalletModal,
  setShowAllocationDrawer,
  setShowSettingsDrawer,
```

Reply:
Unification is definitely desirable, but I would swing it on the opposite. We indicate these booleans in the code everywhere and IMHO it benefits readability greatly.